drivers/sdhc/sdhci_common.c

```diff
@@ -914,14 +1023,36 @@ int sdhci_init_caps(struct sdhci_common *sdhci_ctx, struct sdhc_host_props *prop
 
 int sdhci_init(struct sdhci_common *sdhci_ctx)
 {
+	k_event_init(&sdhci_ctx->irq_event);
```

Comment:
k_event seems to be needed only if irq were enabled. maybe checks should be added?